samples/server/petstore/fsharp-giraffe/OpenAPI.Tests/UserApiTestsHelper.fs

```diff
@@ -40,8 +40,17 @@ module UserApiHandlerTestsHelper =
   let mutable CreateUsersWithArrayInputExamples = Map.empty
   let mutable CreateUsersWithArrayInputBody = ""
 
-  CreateUsersWithArrayInputBody <- WebUtility.HtmlDecode ""
-  CreateUsersWithArrayInputExamples <- CreateUsersWithArrayInputExamples.Add("", CreateUsersWithArrayInputBody)
+  CreateUsersWithArrayInputBody <- WebUtility.HtmlDecode "{
```

Comment:
@nmfisher I notice the use of WebUtility.HtmlDecode here. What if we use `{{{...}}}` (original) instead of `{{...}}` (HTML escaped)?